### PR TITLE
[Docs] function S-U - Fix examples, parameters and notes

### PR DESCRIPTION
**Pull request by seaniepie**

### CODE REVIEW

**Comment by seaniepie** [comment]
That's all of the `<commands>`, `<functions>`, `<control structures>`, `<keywords>`, `<operators>` and `<messages>` repaired that were flagged up by Travis CI. Just a huge list of `<properties>` to go with **852** errors to fix.

File: docs/dictionary/function/uniEncode.lcdoc [hunk 1]
```diff
@@ -112,10 +108,11 @@ added in version 2.0. In previous versions, the <uniEncode>
 Changes:
 The ability to encode text in Polish was added in version 2.1.1.
 
+uniEncode, uniDecode and unicodeText are all deprecated from version 7.0
```
Comment (montegoulding) [inline docs/dictionary/function/uniEncode.lcdoc:112]: @seaniepie this should be a separate `Deprecated:` element

Comment (seaniepie) [reply docs/dictionary/function/uniEncode.lcdoc:112]: Oh cool!

**Comment by montegoulding** [comment]
@livecode-vulcan review ok 311256a

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 311256ac39cab9300d3fda9717f4aeee54c88e73

**Comment by montegoulding** [comment]
@seaniepie for some reason I hadn't noticed that nearly _all_ the properties are listed there. It looks like instead of parameters being listed they document a `value`. The question is should the tests allow that or should they require individual parameters to be described.  For example:

```
set the armedIcon of <button> to {<imageID> | <imageName>}
```

Has two parameters but only one value when you get it. It's probably easier to describe the situation in a single value entry. Perhaps _all_ properties should be documented as:

```
set the armedIcon of <button> to <value>
```

I may have missed something in the documentation about the lcdoc format so I'll take another look for a description of `value` but perhaps @livecodeali has some ideas in this too.

The other issue is it seems so redundant to document the objects here.

**Comment by livecode-vulcan** [comment]
😎 test success 311256ac39cab9300d3fda9717f4aeee54c88e73
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/554)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/559)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/554)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/558)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/563)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/555)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/555)

**Comment by montegoulding** [comment]
Hmm... from the docs:

> When a value is both an input and output value, for example the value of a property, use the Value element. 

This indicates that we should just use `<value>` as the parameter in the property syntax I think... But I want to confirm with the team before we do anything on that.

**Comment by seaniepie** [comment]
`set the armedIcon of <button> to <imageNameOrID>`

**Comment by montegoulding** [comment]
@seaniepie unfortunately that still doesn't help if it is documented in `Value:`. One option is to not do the parameter tests if there is a value element and assume that any named parameters are described in it.

**Comment by seaniepie** [comment]
It will mean a good old fashioned manual check for anything that shows up as a property doc

**Pull request merged**